README.md

```diff
@@ -2781,7 +2781,7 @@ Other Style Guides
     ```
 
   <a name="whitespace--func-call-spacing"></a>
-  - [19.16](#whitespace--func-call-spacing) Enforce spacing between functions and their invocations. eslint: [`func-call-spacing`](https://eslint.org/docs/rules/func-call-spacing)
+  - [19.16](#whitespace--func-call-spacing) Avoid spaces between functions and their invocations. eslint: [`func-call-spacing`](https://eslint.org/docs/rules/func-call-spacing)
```

Comment:
Enforce is correct; we’re enforcing “no spaces”.

Reply:
'Enforce spacing' is little confusing, As It's not clear whether to add spaces or not. Although Example definitely makes this clear or one have to check the eslint doc.

Reply:
The examples and the linter config should make it clear; "enforce no spaces" or "enforce spaces" would both be correct.

Reply:
" Enforce no spaces" or "Avoid spaces" makes it more cleaner than "Enforce spaces". "Enforce no spaces" is good ( If we really want to stick with 'enforce' ). The guide is mostly self explanatory and this change will help to keep it intact.